package.json

```diff
@@ -11,7 +11,9 @@
     "test": "npm run test:lint && npm run test:unit && npm run test:smoke",
     "test:lint": "standard",
     "test:unit": "tape tests/**/*.test.js | faucet",
-    "test:smoke": "tape tests/*.smoketest.js | faucet"
+    "test:smoke": "tape tests/*.smoketest.js | faucet",
+    "snyk-protect": "snyk protect",
+    "prepublish": "npm run snyk-protect"
```

Comment:
I think the intention is better understandable with 'postinstall', even if 'prepublish' might work as well.